prime-router/src/main/kotlin/FakeReport.kt

```diff
@@ -18,19 +18,21 @@ import java.util.concurrent.TimeUnit
     synthetic data, which is data that originates from outside of
     prime and is then shuffled and/or faked
 */
+
+private const val zipCodeData = "zip-code-data"
+
 class FakeDataService {
     fun getFakeValueForElement(
         element: Element,
         context: FakeReport.RowContext,
     ): String {
         val faker = context.faker
-
         // creates fake text data
         fun createFakeText(element: Element): String {
             return when {
                 element.nameContains("name_of_testing_lab") -> "Any lab USA"
                 element.nameContains("lab_name") -> "Any lab USA"
-                element.nameContains("sender_id") -> "" // Allow the default to fill this in
+                element.nameContains("sender_id") -> "${element.default}" // Allow the default to fill this in
```

Comment:
Change this to 
element.default ?: ""
Otherwise you could get a value of "null"

Reply:
When it is element.default ?: "", it still return blank under the send_id column in the output file.  I have to still use "${element.default}"

Reply:
If you are going to leave element.default then at least remove the quotes as they are redundant.

Reply:
Done.

Reply:
Ott, remove the `${}`. That's only used in sting interpolation.